### PR TITLE
Improve comments for Intellisense 

### PR DESCRIPTION
**Pull request by networkfusion**
## Description
Improves the class documentation.

## Motivation and Context
The SNTP class could be improved.

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [x] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [ ] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

File: nanoFramework.Networking.Sntp/Sntp.cs [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
         /// <summary>
         /// Starts the SNTP client.
-        /// After this method returns a periodic call to the set SNTP servers is performed providing time synchronization.
-        /// The system time is updated immediately after each successful execution.
-        /// The time synchronization occurs on a hourly rate.
+        /// After this method returns, a periodic call to the set SNTP servers is performed providing time synchronization.
         /// </summary>
-        /// <remarks>This method is specific to nanoFramework.</remarks>
+        /// <remarks>
+        /// The system time is updated immediately after each successful execution.
```
Comment (josesimoes) [inline nanoFramework.Networking.Sntp/Sntp.cs:23]:
```suggestion
        /// <para>
        ///  The system time is updated immediately after each successful execution.
        /// </para>
```

File: nanoFramework.Networking.Sntp/Sntp.cs [hunk 2]
```diff
@@ -64,8 +69,13 @@ public class Sntp
         }
 
         /// <summary>
-        /// Gets, sets the address of the SNTP server 2.
+        /// Gets, sets the host address of the backup SNTP server.
+        /// By default it uses the server '1.pool.ntp.org'
```
Comment (josesimoes) [inline nanoFramework.Networking.Sntp/Sntp.cs:73]:
```suggestion
        /// By default it uses the server '1.pool.ntp.org'.
```

File: nanoFramework.Networking.Sntp/Sntp.cs [hunk 3]
```diff
@@ -64,8 +69,13 @@ public class Sntp
...
         /// </summary>
+        /// <remarks>
+        /// For a list of some public NTP servers, see this link :
```
Comment (josesimoes) [inline nanoFramework.Networking.Sntp/Sntp.cs:76]:
```suggestion
        /// For a list of some public NTP servers, see this link:
```

File: nanoFramework.Networking.Sntp/Sntp.cs [hunk 4]
```diff
@@ -74,5 +84,41 @@ public class Sntp
             [MethodImpl(MethodImplOptions.InternalCall)]
             set;
         }
+
+        // /// <summary>
```
[the remark this answers is not in the document]
Comment (networkfusion) [reply nanoFramework.Networking.Sntp/Sntp.cs:92]: They are new, but I was unsure of what I need to do to get them to work in unmanaged code.
For the moment, I will just remove them from this PR and create a new one for that.

**Review by Ellerbach — APPROVED** [review]
Much better. thanks!

**Pull request merged**